R/pkg/inst/tests/testthat/test_utils.R

```diff
@@ -17,6 +17,9 @@
 
 context("functions in utils.R")
 
+# Ensure Spark is installed
+sparkCheckInstall()
```

Comment:
I just tested this by putting SparkR:::checkInstall in run-all.R (before calling test_package) and that seems to do the trick on a custom 2.1.0 build ! 

@felixchueng when you get a chance can you update the PR with that ? The only thing that I'm concerned about is calling a private function from `run-all.R` - We could either export this function or move some of this functionality into `install.spark`